### DISCUSSION TITLE
Waiting for the next release

### DISCUSSION THREAD

**Discussion by fidgi** [loopbackio/loopback-next#8217] [Q&A] [answered]
Hi,
When do you plan to issue a new release ?
I'm waiting for it in order to get rid of the 2 moderate severity vulnerabilities from swagger-ui-dist dependency

regards,

**Comment by dhmlau** [comment]
@fidgi, will discuss with maintainers if anything blocking us from publishing a release this week. 
Was meant to publish before the holidays/new year, but got sidetracked by vacation. :)

**Comment by achrinza** [comment, marked as answer]
A release has been published: https://github.com/loopbackio/loopback-next/commit/4a8886403cd9cb3ae529cb69712f077623cba2c2
